content/en/apps/guides/hosting/ec2-setup-guide.md

```diff
@@ -1,110 +1,123 @@
+1. Restarting processes
+    - [How to access container, retrieve logs, restart services]({{< ref "core/guides/docker-setup#helpful-docker-commands" >}})
+    - [MedicOS service management scripts](https://github.com/medic/medic-os#user-content-service-management-scripts)
 
-#### 4a. RDS help
-- [Amazon user guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html)
+1. Investigating logs
+    - [Helpful docker commands]({{< ref "core/guides/docker-setup#helpful-docker-commands" >}}) (includes getting shell on containers)
+    - Inside container, all appropriate logs can be found in: `/srv/storage/<service_name>/logs/*.log`
 
-## Restore from backup and monitoring
-#### 1. Configure backups
-- [EBS Snapshot Lifecycle Manager](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snapshot-lifecycle.html)
+1. Upgrading the container
+    - Backup all data (EBS) 
+    - Log into container and stop all services
+    - DO NOT REMOVE `/srv/storage/medic-core/`, `/srv/settings/medic-core/couchdb/local.ini`, wipe all other files in `/srv` [Note: Make script publicly accessible]
+    - [Change the image tag to the newest image release version]({{< ref "core/guides/docker-setup#use-docker-compose" >}})
+    - [Change image tag in docker-compose file]({{< ref "core/guides/docker-setup#use-docker-compose">}})
+    - Launch new containers with appropriate COUCHDB_ADMIN_PASSWORD & HA_PASSWORD environment variables
 
-#### 2. Restoring from backup
-- Create volume from snapshot
-- Tag appropriately for backups
-- Mount volume to docker server
+1. Upgrading the webapp
+    - Use Admin GUI page
+    - [CLI via horticulturalist]({{< ref "apps/guides/hosting/self-hosting#links-to-medic-documentation-for-horticulturalist-for-upgrades" >}})
 
-#### 3. Process supervison
-- supvisorctl
-- /boot/supervisor-inspect
+1. RDS help
 
-#### 4. Monitoring
-- [Cloudwatch](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-cloudwatch.html)
+    - [Amazon user guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Welcome.html)
 
-## Own Infrastructure Setup
-#### 1. Team to set up own infrastructure based on best-practices and recommendations above
+## Backups 
 
+1. Configure backups
+    - [EBS Snapshot Lifecycle Manager](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/snapshot-lifecycle.html)
 
+1. Restoring from backup
+    - Create volume from snapshot
+    - Tag appropriately for backups
+    - Mount volume to docker server
 
+## Process supervision
+- `supvisorctl`
+- `/boot/supervisor-inspect`
 
+## Increasing disk size
 
+* Stop medic: `sudo supervisorctl stop medic`
+* Go to EBS in AWS and take a snapshot of the volume.
+* Modify the volume size (Increase it by 2x preferably). Wait until the modification succeeds.
+* [Make the instance recognize the additional space](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/recognize-expanded-volume-linux.html)
+* Turn medic back on: `sudo supervisorctl start medic`
```

Comment:
It's a good idea to add when the disk should be resized. If you wait until the disk is almost full, it's too late. E.g. CouchDB runs compaction on schedule and compaction fails if the free disk space is low. If compaction fails, that in turn means the disk fills up more rapidly. So basically monitor for if/when compaction starts failing.

Reply:
Ah - totally makes sense!  

The content I'm editing here is [what we had published already](https://docs.communityhealthtoolkit.org/apps/guides/hosting/self-hosting/), so I didn't write it.  More importantly, AWS is totally unknown to me.  

Could I bother you to give me an explicit suggestion to improve these steps so we can ensure it's technically accurate?

Reply:
Let's set free disk space to be 40% or more.

Reply:
Gotcha- thanks for the specifics. Added a note about this in b7eaeca